src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/repository/BitbucketCloudRepository.java

```diff
@@ -48,7 +48,10 @@
     private BitbucketCloudRepositoryOwner owner;
 
     @JsonProperty("updated_on")
-    private String updatedOn;
+    private Date updatedOn;
+
+    @JsonProperty("slug")
```

Comment:
This shouldn't be needed? The field name already follows the name convention.

Reply:
Good catch, I forgot to use it. slug is the repo id. Ideally we should be using uuid but then it doesn't work with our case.

See https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D.

We should simply return slug in getRepositoryName() instead of split fullName. 

```
        if (this.fullName != null) {
            return this.fullName.split("/")[1];
        }
```